packages/@extension-kid/notification-service/__tests__/NotificationService-test.ts

```diff
@@ -0,0 +1,85 @@
+import { Container } from "@extension-kid/core";
+import notificationServiceContainerModuleFactory, {
+  getExtensionModule,
+  Notification,
+  NotificationService,
+  NotificationServiceType,
+  NotificationServiceExtensionInterface
+} from "../index";
+import { injectable } from "inversify";
+
+const DebugNotification = Symbol("DebugNotification");
+
+const DebugSymbol = Symbol("Debug");
+
+@injectable()
+class DebugNotificationExtension
+  implements NotificationServiceExtensionInterface {
+  id = DebugSymbol;
+
+  supportedNotifications(): symbol[] {
+    return [DebugNotification];
+  }
+
+  push(notification: Notification) {
+    /* tslint:disable */
```

Comment:
tslint does not seem to have a problem here, does it?

Reply:
I wonder if we disabled the no-logs rule 🤔

Reply:
i'm so sorry. some fault in my editor-setup.